eras/babbage/impl/src/Cardano/Ledger/Babbage/TxBody.hs

```diff
@@ -755,6 +756,7 @@ instance CC.Crypto c => ShelleyMAEraTxBody (BabbageEra c) where
   {-# INLINE mintTxBodyL #-}
 
   mintValueTxBodyF = mintValueBabbageTxBodyF
+  {-# INLINE mintValueTxBodyF #-}
```

Comment:
you meant `INLINE` and not `INLINEABLE`, right?

Reply:
Yes. This is a very small function, it is good to inline it :)
`mintValueTxBodyF = mintValueBabbageTxBodyF`